packages/data-point/lib/reducer-object/__snapshots__/factory.test.js.snap

```diff
@@ -6,24 +6,19 @@ Array [
     "path": Array [
       "a1",
     ],
-    "transform": TransformExpression {
-      "reducers": Array [
-        ReducerPath {
-          "asCollection": true,
-          "body": [Function],
-          "name": "a",
-          "type": "ReducerPath",
-        },
-      ],
-      "typeOf": "TransformExpression",
+    "transform": ReducerPath {
+      "asCollection": true,
+      "body": [Function],
+      "name": "a",
+      "type": "ReducerPath",
```

Comment:
Before, every reducer was wrapped in a TransformExpression, even when there was only one of them. Now if there's just one reducer, it's not wrapped in a ReducerList (which is why a TransformExpression is being replaced with a ReducerPath)